kiwi/volume_manager/btrfs.py

```diff
@@ -503,76 +502,22 @@ def _set_default_volume(self, default_volume):
             'Failed to find btrfs volume: %s' % default_volume
         )
 
-    def _xml_pretty(self, toplevel_element):
-        xml_data_unformatted = ElementTree.tostring(
-            toplevel_element, 'utf-8'
-        )
-        xml_data_domtree = minidom.parseString(xml_data_unformatted)
-        return xml_data_domtree.toprettyxml(indent="    ")
-
     def _create_snapper_quota_configuration(self):
         root_path = os.sep.join(
             [
                 self.mountpoint,
                 f'{self.root_volume_name}/.snapshots/1/snapshot'
             ]
         )
-        snapper_default_conf = Defaults.get_snapper_config_template_file(
-            root_path
+        # snapper requires an extra parent qgroup to operate with quotas
+        Command.run(
+            ['btrfs', 'qgroup', 'create', '1/0', self.mountpoint]
```

Comment:
This is again something that snapper should do rather than us.

Reply:
Not so sure about this, I don´t know enough about this use case for a proper opinion, but at a glance I am not convinced this is on snapper responsibilities. Snapper _only_ manages subvolumes in a CRUD fashion and it expects a btrfs filesystem there ready to be used. We are just telling to snapper which quota group to use on the existing filesystem.